portal/catalog.ipynb

```diff
@@ -0,0 +1,2615 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "title: \"Catalog\"\n",
+    "format:\n",
+    "    html:\n",
+    "        toc: true\n",
+    "        theme: darkly\n",
+    "        page-layout: full\n",
+    "        anchor-sections: true\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a \n",
+    "    target=\"_blank\" \n",
+    "    href=\"https://colab.research.google.com/github/davidgasquez/gitcoin-grants-data-portal/blob/main/notebooks/sandbox.ipynb\"> \n",
+    "    <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
+    "</a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Following parquet files can be accessed from `ipfs`, see the **home** page for instructions.\n",
+    "\n",
+    "Raw files are direct result of API fetch, while processed files have been cleaned and normalized. \n",
+    "\n",
+    "Current convention for processed files is to store `address` address in lowercase format. Columns are renamed to `snake_case`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| echo: false\n",
+    "import os\n",
+    "\n",
+    "import pandas as pd\n",
+    "import pyarrow.parquet as pq\n",
+    "from IPython.display import display, Markdown"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {
+    "tags": [
+     "hide_input"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Processed Files:\n",
+      "application_answers.parquet\n",
+      "chain_metadata.parquet\n",
+      "passport_scores.parquet\n",
+      "projects.parquet\n",
+      "round_applications.parquet\n",
+      "round_votes.parquet\n",
+      "rounds.parquet\n",
```

Comment:
Could we print this in Markdown with links to their table headers?

Reply:
Good idea, will do.

Reply:
Done. Two way links - clicking will bring you back to file listing.